e2-studio-star-rx72n-firmware/libs/rx_usb_comm/src/rx_usb_comm.c

```
@@ -479,11 +479,11 @@ static rx_err_t internal_find_sync(const rx_usb_comm_handle_t* handle, int32_t*
   }
 
   /* Extract sync word bytes using shared constants from rx_frame.h */
-  const uint8_t sync_high = (uint8_t)(k_frame_sync_word >> k_rx_be16_high_shift);
   const uint8_t sync_low  = (uint8_t)(k_frame_sync_word & k_rx_byte_mask);
```

Clang (14.0.6)

[warning] variable 'sync_low' is not initialized

(cppcoreguidelines-init-variables)